templates/zerver/integrations.html

```diff
@@ -553,6 +559,31 @@
       <img class="screenshot" src="/static/images/integrations/bitbucket/002.png" />
     </div>
 
+    <div id="bitbucket2" class="integration-instructions">
+
+     <p>Zulip supports both Git and Mercurial notifications from Bitbucket.</p>
+
+     <p>First, create the stream you'd like to use for Bitbucket notifications,
+     and subscribe all interested parties to this stream. The integration will
+     use the default stream <code>bitbucket</code> if no stream is supplied in the
+     hook; you still need to create the stream even if you are using this
+     default.</p>
+
+     <p><code>{{ external_api_uri }}/v1/external/bitbucket2?api_key=abcdefgh&amp;stream=bitbucket</code></p>
+
+     <p>where <code>api_key</code> is the API key of your Zulip bot,
+       and <code>stream</code> is the stream name you want the
+       notifications sent to.</p>
+
+     <p>Next, from your repository's web page, go to the Settings page and choose Webhooks on the left-hand side.
+     Click <code>Add webhook</code>, set title, url (the url you created above). Remember about check active checkbox.
```

Comment:
"Remember about check active checkbox." - do you mean, "Remember to click the 'active' checkbox"? If so, that's a clearer way to say it.